app/models/api/suggestions.php

```diff
@@ -56,6 +56,14 @@
     return $value;
 };
 
+if (empty($get_option('max_results')) || $get_option('max_results') == 0) {
```

Comment:
The right approach is to evaluate `$max_results` inside the called method, not duplicate the call. The function is also very similar to `getTranslationMemoryResults`, so you can take inspiration for the check.

Also note that `$get_option('max_results')` can't really be empty.